genome_designer/main/tests/test_genome_finish.py

```diff
@@ -131,7 +136,12 @@ def _run_genome_finish_test(self, data_dir):
                 new_ref_genome, Dataset.TYPE.REFERENCE_GENOME_FASTA
                 ).get_absolute_location()
 
-        self.assertTrue(are_fastas_same(target_fasta, new_ref_genome_fasta))
+        fastas_same, indexes = are_fastas_same(
+                target_fasta, new_ref_genome_fasta)
+
+        if not fastas_same:
+            raise Exception(
```

Comment:
Rather than `raise Exception`, use `self.assertTrue()`. If you need the message you can pass it to this method, e.g. https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertTrue